src/components/RuntimeManager.js

```diff
@@ -183,6 +229,11 @@ export default class RuntimeManager extends PureComponent {
           })
         ])
       })
+    } else if (prevApp && prevApp.status !== 'ERROR' && app && app.status === 'ERROR' && !_.includes(app.appName, errorNotifiedApps.get())) {
```

Comment:
I think should be `else if (app?.status === 'ERROR' && !_.includes(app.appName, errorNotifiedApps.get()))`. I think it's important to show the error on page load, or if they navigate away when the error occurs.

Reply:
ah yeah i like that idea, with the runtime it saves it to the little error icon so they can reopen it but galaxy doesnt have that